lib/mpl_toolkits/axisartist/axislines.py

```diff
@@ -311,10 +305,8 @@ def __init__(self, axes):
         super().__init__()
         self.axes = axes
 
-    @_api.delete_parameter(
-        "3.9", "nth_coord", addendum="'nth_coord' is now inferred from 'loc'.")
     def new_fixed_axis(
-            self, loc, nth_coord=None, axis_direction=None, offset=None, axes=None):
+            self, loc, axis_direction=None, offset=None, axes=None):
```

Comment:
The remaining parameters must be keyword-only now.
```suggestion
            self, loc, *, axis_direction=None, offset=None, axes=None):
```